src/modules/ethereum/transactions/types.ts

```diff
@@ -12,7 +12,7 @@ export interface TransactionObject {
 
 export type StoredTransactions = Array<TransactionObject>;
 
-interface Warning {
+export interface InterpretWarning {
```

Comment:
Where is this used?

Reply:
The idea was to display these warnings in `TransactionWarnings` component (before I realized there is no interpretation warnings yet), I'll revert these changes for now